src/entities/course.entity.ts

```diff
@@ -13,9 +13,15 @@ export class CourseEntity extends BaseEntity {
   @Column()
   slug: string;
 
-  @Column()
+  @Column({
+    unique: true,
+    nullable: true,
+  })
   active: SIMPLYBOOK_STORY_STATUS_ENUM;
```

Comment:
this isnt simplybook status right? it's storyblok
should probably rename `active` to `status` now we've made this change